CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ set( with-libneurosim OFF CACHE STRING "Build with libneurosim [default=OFF]. Op
 set( with-music OFF CACHE STRING "Build with MUSIC [default=OFF]. Optionally give the directory where MUSIC is installed." )
 set( with-sionlib OFF CACHE STRING "Build with SIONlib [default=OFF]. Optionally give the directory where sionlib is installed." )
 set( with-boost ON CACHE STRING "Build with Boost [default=ON]. To set a specific Boost installation, give the install path." )
+set( with-sonata ON CACHE STRING "Find a HDF5 library. To set a specific HDF5 installation, set install path. [default=ON]" )
```

Comment:
This option should be `with-hdf5` because it controls if and optionally which HDF5 library is included. We may later use HDF5 in other contexts than Sonata and then it will become confusing to have the option named for Sonata.

Reply:
I have now replaced option names and messages to use HDF5 instead of SONATA.